frontend/src/e2e-playwright/pages/employee/child-information.ts

```diff
@@ -0,0 +1,90 @@
+import { Page } from 'playwright'
+import { RawElement } from 'e2e-playwright/utils/element'
+
+export type Collapsible = 'dailyServiceTimes'
+
+export default class ChildInformationPage {
+  constructor(private readonly page: Page) {}
+
+  readonly #collapsibles: Record<Collapsible, RawElement> = {
```

Comment:
is this `#variable` some syntax I haven't seen before or some kind of naming convention?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#private_field_declarations

It was used in some other page model, so I continued to use it.